### ISSUE TITLE
Linking video timecodes to data feeds - issues

### ISSUE THREAD

**Issue by bquinn** [iptc/sport-schema#148]
At the call last week, we discussed issues around linking video timecodes to sports data feeds.

We would like to have a way of linking associated video, but we can't do it automatically based on timecodes of played-out video because many sports broadcasts do not air in real time. At best, there is often 2 or 3 seconds latency between the action on the pitch and the video being transmitted. At worst, the video might be played as a catch-up, particularly in multi-location games like golf or athletics where the broadcast footage may move back and forth to cover action that was happening simultaneously.

So we should have some mechanism for saying "this video depicts this action", but using a simple timecode probably isn't the way to do it.

In summary, "actual time" and "broadcast time" may be quite different. This could also be true because the broadcaster is in a different location.

Relateed to #149
